packages/table/src/table.tsx

```diff
@@ -492,7 +492,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     // time. it serves as a signal that we can switch to batch rendering.
     private didCompletelyMount = false;
 
-    public constructor(props: ITableProps, context?: any) {
+    public constructor(props: ITableProps & { children: React.ReactNode }, context?: any) {
```

Comment:
Worth making a new props interface in `core/.../props.ts`? This is used in a few places.

e.g.
```tsx
export interface IChildrenProps {
    children: React.ReactNode;
}
```

Reply:
no, I removed it